### PR TITLE
[Security] Getting Started with Nessus Scanner

### LINKED ISSUE #5934: [Security] Getting started with Nessus scanner
#### Proposed title of article
Getting started with Nessus Scanner

#### Proposed article introduction
In this tutorial, I will be introducing you to the Nessus scanner and how to be a good Pen-tester using Nessus.
Nessus is a Web vulnerability scanner that allows you to scan your server/Website for vulnerabilities and provide tips on how to fix them. This is open-source software that is available for all, it is available on Windows but it comes inbuilt in Kali-Linux which makes things even easier.

#### Key takeaways 
The reader should be able to
- Install Nessus and know how to use it.

#### Article quality
Most sites covering the same topic only show you how the installation works and how to scan for vulnerabilities, but how to fix this vulnerabilities with nessus they don't show you.
So I will hold you by the hand and show you different ways to help secure a real site with Nessus after a successful scanning is done. You will be using Nessus to find, fix and update servers to increase your security levels.

### PR DESCRIPTION
**Pull request by Adespscientist**
> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5934

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.
No

### CODE REVIEW

**Comment by srishilesh** [comment]
Hello @Adespscientist ,

Welcome to EngEd community!

Your PR has entered our reviewing queue. Please wait until a peer-reviewer pings you for the revisions to be made.

Meanwhile, we suggest you to go through the published articles or approved PRs to see how the other authors have formatted their articles to avoid any potential closure of PRs.

Thanks.

**Comment by srishilesh** [comment]
@Adespscientist 

Can you please link this PR to an issue?

**Comment by Adespscientist** [comment]
> @Adespscientist 
> 
> Can you please link this PR to an issue?

Thank you.
I have linked this PR to an issue.

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

**Comment by Bayler** [comment]
@Adespscientist Did you forget to upload your author bio?

**Comment by Adespscientist** [comment]
> @Adespscientist Did you forget to upload your author bio?

I have updated the author bio sir.
Thank you

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+author: adeshina-peter
+date: 2022-01-27T00:00:00-17:00
+topics: [Security]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/getting-started-with-nessus-scanner/hero.jpg
+    alt: Getting Started with Nessus Scanner Hero Image
+---
+**Nessus Tenable** is a kali-Linux open-source software common among penetration testers, used to provide a variety of security fixes. You are going to be using this tool quite frequently while you work as a pen-tester by scanning for potential vulnerability on a Server/Website.
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:17]:
```suggestion
**Nessus Tenable** is a Kali-Linux open-source software common among penetration testers, used to provide a variety of security fixes. You will use this tool quite frequently while you work as a pen-tester by scanning for potential vulnerability on a server/website.
```

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+<!--more-->
+**Nessus** scanner is a professional tool that is widely used among hackers and penetration testers. The solution you get assists you to perform high-speed asset discovery, target profiling, malware detection, sensitive data discovery and so many more.
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:19]:
```suggestion
**Nessus** scanner is a professional tool widely used among hackers and penetration testers. The solution you get assists you to perform high-speed asset discovery, target profiling, malware detection, sensitive data discovery, and so many more.
```

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+**Tenable** delivers updated plug-ins that provide you with the latest information to customers within 24 hours after a vulnerability has been leaked.
+**Nessus Tenable** grant you the privileges to scan basic networks and servers for free, and also provide you with how **critical**,**medium** or how **low** a vulnerability is and helps provide descriptions and solutions for a fix.
+
+Most sites that talk about Nessus only point out how Nessus is being installed and how it is been used, But how to fix a real vulnerability is hardly in the scope of what they teach. I will show you how to fix a common vulnerability and also guide you through important Nessus keywords.
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:23]:
```suggestion
Most sites that talk about Nessus only point out how to install and use Nessus. But, how to fix a real vulnerability is hardly in the scope of what they teach. We will show you how to fix a common vulnerability and guide you through important Nessus keywords.
```

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+- Understand procedures to take in fixing vulnerabilities.
+
+### Advantages of Nessus Scanner
+- It helps you Identify vulnerabilities before attackers can take advantage of them.
+
+- It helps you find misconfigurations like missing patches, mail-relay, etc.
+
+- Capturing of data and showing results reporting impact on the network 
+
+If you don't have Kali-Linux installed on your pc, Don't worry I will show you how to quickly set this up.
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:45]: Avoid using the word, "I" in your tutorials. The focus should always be on the reader. Please use 'we' instead and add commas where necessary. i.e, don't worry, .......

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+
+- [Kali Linux iso](https://www.kali.org/get-kali/#kali-platforms)
+
+#### Download Vmware (virtual box)
+
+After successfully downloading the Kali Linux iso, you can now download the VMware virtual box using the link below:
+
+- [Vmware virtual box](https://www.vmware.com/products/workstation-player/workstation-player-evaluation.html).
+
+Follow the [installing Vmware](https://forums.tomshardware.com/faq/how-to-install-vmware-workstation-in-windows-10.3273363/) tutorials to successfully install the VMware virtual box.
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:61]:
```suggestion
Follow the [installing Vmware](https://forums.tomshardware.com/faq/how-to-install-vmware-workstation-in-windows-10.3273363/) tutorials to install the VMware virtual box.
```

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+
+After Successfully installing Kali-Linux in a Virtual-Box, You can now Start Kali by
+Opening the virtual box, select the Kali Linux option, and click the play button to start your Kali Linux.
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:64]: Avoid adding capital letters unnecessarily.

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+
+ - Holding the **cltr+alt+T** buttons simultaneously or
+   - Using the GUI (Graphical User Interface), go to the applications section and double click on the terminal option.
+
+Now run the command below on the terminal to update and upgrade your Kali Linux.
+
+```bash
+    $ sudo apt-get upgrade && update -y
+```
+Now let's change our directory into where Nessus tenable is been installed.
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:88]:
```suggestion
Now, let's change our directory to where Nessus tenable is installed.
```

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+
+```bash
+    $ cd Downloads
+```
+Use the command below to list  all files in `Downloads` then copy the downloaded file name
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:93]:
```suggestion
Use the command below to list all files in `Downloads`, then copy the downloaded file name.
```

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+    $ ls
+``` 
+Use the command below to grab the package to download it and install it.
+
+```bash
+    $ dpkg -i Nessus-10.0.2-ubuntu910_amd64.deb
+```
+![dpkg](dpkg-nessus.png)
+
+You can now start Nessus scanner by typing the following into your terminal.
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:105]:
```suggestion
You can now start the Nessus scanner by typing the following into your terminal.
```

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+Then Navigate to the following URL in your browser to start Nessus Scanner.
+
+- [Nessus localhost](https://kali:8843/) to configure your scanner
+
+- Default port to run Nessus is **`8834`**
+
+If you see connection not secured,  click *advance* Accept risk and continue.
+
+### Step 3:
+- And here you have Nessus already on your screen, It going to compile plugins and it going to take a little bit of time so go ahead and let it finish. When it does go ahead and click `Nessus Essentials` and click continue.
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:119]: Please check your grammar use.

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 11]
```diff
@@ -0,0 +1,240 @@
...
+
+- Now provide Your name and a valid Email to receive an activation code.
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:121]:
```suggestion
- Now, provide your name and a valid email to receive an activation code.

```

**Comment by Bayler** [comment]
@Adespscientist I will be closing this PR. It has a lot of grammatical errors. Go through the few suggestions I've made to help you understand where you're going wrong. Also, you can leverage grammarly premium to help correct your grammar. Finally, I really recommend you take the tech writing course provided by Google: https://developers.google.com/tech-writing
Let me know when do so that we can reopen this PR. As it is, the article doesn't match our publishing level.

cc: @section-engineering-education/content-moderator-team

**Comment by Adespscientist** [comment]
Thank you.. I will work on it asap

**Comment by Adespscientist** [comment]
I have made changes to the PR. Thank you sir

**Comment by Petekavinsky** [comment]
@hectorkambow, @WanjaMIKE, Please what is the update on my PR. Thankyou

**Comment by Adespscientist** [comment]
Hello @WanjaMIKE , please what is the update on my PR.

**Comment by hectorkambow** [comment]
jumping on this one

**Comment by WanjaMIKE** [comment]
Just finishing up @hectorkambow

**Comment by hectorkambow** [comment]
> Just finishing up @hectorkambow

thank you for letting me know 🚀  

I'll jump on another one.

**Comment by WanjaMIKE** [comment]
There is this image *new-scan.png*. I do not see where it is cited @Adespscientist

**Comment by Adespscientist** [comment]
@WanjaMIKE, I have deleted the image "new-scan".

**Comment by WanjaMIKE** [comment]
Merged #6155 into master

**Comment by Adespscientist** [comment]
@WanjaMIKE, @Bayler, @hectorkambow thank you for reviewing my PR

**Pull request merged**